Okta.Xamarin/Okta.Xamarin/IOidcClient.cs

```diff
@@ -12,10 +12,11 @@ namespace Okta.Xamarin
 	/// </summary>
 	public interface IOidcClient
 	{
+		object ReturnContext { get; set; }
 		/// <summary>
-		/// The configuration for this Client.  Must be set in the constructor.
+		/// The configuration for this Client.
 		/// </summary>
-		IOktaConfig Config { get; }
+		IOktaConfig Config { get; set; }
```

Comment:
Shouldn't the config be immutable?

Reply:
Order of operations issues and a need to be able to construct a client without parameters for both Android and iOS necessitated this setter be made public.  I can plan to make it "lazy immutable" by ensuring it is set only once where appropriate.